src/blocks/Transaction.vue

```diff
@@ -134,6 +134,10 @@
       <div v-if="type === 'CPK' && cpkStatus === true" class="_text-center _margin-top-1">Your account is already activated</div>
 
       <div v-if="error" class="errorText _text-center _margin-top-1" data-cy="transaction_error_text">{{ error }}</div>
+      <div v-if="chosenToken && mainToken === 'L2-NFT' && !nftExists && !nftExistsLoading" class="errorText _text-center _margin-top-1">
```

Comment:
Implode it into computed property. This should be done any time you're using more then 2 conditions directly in the code